components/Modal/Modal.js

```diff
@@ -38,22 +38,23 @@ function Modal({
   }
 
   return (
-    <ReactModal
-      portalClassName={ModalStyles.Modal}
-      className={classNames(CardStyles.Card, className)}
-      contentLabel={screenReaderLabel}
-      isOpen={isOpen}
-      onRequestClose={onRequestClose}
-      shouldCloseOnOverlayClick={shouldCloseOnOverlayClick}
-    >
-      <CloseButton onClick={onRequestClose} />
-      <div
-        className={childrenClassName || ModalStyles.scrollableContainer}
-        data-testid={MODAL_CONTENT}
-      >
-        {children}
-      </div>
-    </ReactModal>
+    <Dialog.Root defaultOpen={false} open={isOpen}>
```

Comment:
[Notice in the Radix Dialog documentation](https://www.radix-ui.com/docs/primitives/components/dialog), the Dialog button has a bunch of aria properties. I don't actually know if these are needed, since the [WAI modal example](https://www.w3.org/TR/wai-aria-practices-1.2/examples/dialog-modal/dialog.html) has a button _without_ aria attributes. So, I've asked the Radix team. Let me get an answer from them before I give 👍🏼 

```html
<button type="button" aria-haspopup="dialog" aria-expanded="false" aria-controls="radix-30" data-state="closed" class="c-hYXuTU c-hYXuTU-inZbjw-variant-violet">Edit profile</button>
```

Reply:
They strongly encourage using the trigger, but it isn't required to be accessible.